benchmarks/dynamo/common.py

```diff
@@ -2969,7 +2969,7 @@ def get_example_inputs(self):
 def process_entry(rank, runner, original_dir, args):
     args.rank = rank
     with maybe_init_distributed(
-        args.multiprocess,
```

Comment:
@H-Huang 
> I'm confused how come parent process is initializing distributed when it should just be spawning the child processes (calling mp.spawn and waiting for child processes to finish)?

I changed this in #108438, but didn't run into the failure because I didn't test the parent run, so I'm adding back the `args.only` check to only init distributed in child runs

Reply:
Sounds good